packages/terra-hyperlink/src/Hyperlink.scss

```diff
@@ -0,0 +1,97 @@
+@import '~terra-mixins';
+
+:local {
+  .hyperlink {
+    background-color: transparent; // Remove the gray background on active links in IE 10
+    color: var(--terra-link-color, #0059a8);
+    display: inline-block; // prevents variant icon from breaking onto its own line
+    outline: none;
+    text-decoration: var(--terra-link-underline, none);
+
+    &:visited,
+    &.is-visited {
+      color: var(--terra-link-visted-color, #0059a8);
+    }
+
+    &:hover,
+    &.is-hovered {
+      color: var(--terra-link-hover-color, #001f67);
```

Comment:
Some of our components change the background when hovered over, such as our documentation site. Perhaps we should give consumers that option here as well?

Reply:
Added ability to set background color on hover via theming variable: https://github.com/cerner/terra-core/commit/5d1a6de0b4dd80fbbb620354ee5b823764386a5b